tests/gpflux/layers/test_dedup_trackable_layer.py

```diff
@@ -100,6 +101,7 @@ def _size_q_mu(num_inducing, output_dim):
 }
 
 
+@pytest.mark.skipif(Version(tf.__version__) <= Version("2.3"), reason="Do not run with TF-2.2")
```

Comment:
It looks like the "reason" and "if-statement" don't match. 
```suggestion
@pytest.mark.skipif(Version(tf.__version__) < Version("2.3"), reason="Do not run with TF-2.2")
```
or
```suggestion
@pytest.mark.skipif(Version(tf.__version__) <= Version("2.3"), reason="Do not run with TF-2.2 and TF-2.3")
```